backend/app/db_accessors/accessor.js

```diff
@@ -0,0 +1,44 @@
+import Connection from "../db/connection.js";
+import Blog from "../models/blog.js";
+
+export default class Accessor {
+    static async getAllBlogs() {
+        try {
+            await Connection.open("teaching_proj");
+            const messages = await Message.find({});
+            return messages;
+        } catch (e) {
+            console.log("Something bad will happen");
+        }
+    }
+
+    static async postBlog(blogInfo) {
+        try {
+            await Connection.open("teaching-proj");
+            const msge = await Blog.create(blogInfo);
+            return msge;
+        } catch (e) {
+            console.log("Failed");
+        }
+    }
+
+    static async updateBlogById(id, updatedInfo) {
+        try {
+            await Connection.open("teaching-proj");
+            const msge = await Blog.findOneAndUpdate({ _id: id }, { $set: updatedInfo }, { new: true, runValidators: true });
```

Comment:
If you're not finding by index, you can use findByIdAndUpdate, for this specific use case.

https://mongoosejs.com/docs/api/model.html#Model.findByIdAndUpdate()

If you end up searching by, Title for example, and all Titles are unique, then you should use findOneAndUpdate